docs/standard/native-interop/best-practices.md

```diff
@@ -182,16 +182,16 @@ The following types are the same size on 32-bit and 64-bit Windows, despite thei
 | 16    | `WORD`           | `unsigned short`     | `ushort` |                                      |
 | 16    | `ATOM`           | `unsigned short`     | `ushort` |                                      |
 | 32    | `INT`            | `int`                | `int`    |                                      |
-| 32    | `LONG`           | `long`               | `int`    |                                      |
-| 32    | `ULONG`          | `unsigned long`      | `uint`   |                                      |
-| 32    | `DWORD`          | `unsigned long`      | `uint`   |                                      |
+| 32    | `LONG`           | `long`               | `int`    |  See `CLong` and `CULong` below.     |
+| 32    | `ULONG`          | `unsigned long`      | `uint`   |  See `CLong` and `CULong` below.     |
+| 32    | `DWORD`          | `unsigned long`      | `uint`   |  See `CLong` and `CULong` below.     |
 | 64    | `QWORD`          | `long long`          | `long`   |                                      |
 | 64    | `LARGE_INTEGER`  | `long long`          | `long`   |                                      |
 | 64    | `LONGLONG`       | `long long`          | `long`   |                                      |
 | 64    | `ULONGLONG`      | `unsigned long long` | `ulong`  |                                      |
 | 64    | `ULARGE_INTEGER` | `unsigned long long` | `ulong`  |                                      |
-| 32    | `HRESULT`        | `long`               | `int`    |                                      |
-| 32    | `NTSTATUS`       | `long`               | `int`    |                                      |
+| 32    | `HRESULT`        | `int`                | `int`    |                                      |
```

Comment:
This is technically incorrect since this column is called `C (Windows)` and HRESULT is really `long` on Windows. It may be best to delete the `C (Windows)` column. I do not think that it is helping.